lib/components/Input/base/base.scss

```diff
@@ -0,0 +1,349 @@
+@import "../../../styles/quill/spacing.scss";
+@import "../../../styles/quill/font.size.scss";
+@import "../../../styles/quill/color.scss";
+@import "../../../styles/quill/color.semantic.scss";
+@import "../../../styles/quill/color.opacity.scss";
+@import "../../../styles/quill/motion.scss";
```

Comment:
Could you please update imports? As I understand, now they can be written as @quill/...

Reply:
oh yeah. I didn't commit it. thanks for noticing